### PR TITLE
Update tests with broken GCS links

### PR DESCRIPTION
**Pull request by nnegrey**
GCS files no longer exist, updated to general GCS location and updated tests accordingly

### CODE REVIEW

**Comment by nnegrey** [comment]
Fixes: https://github.com/GoogleCloudPlatform/java-docs-samples/issues/1786

File: language/analysis/src/test/java/com/google/cloud/language/samples/AnalyzeBetaIT.java [hunk 1]
```diff
@@ -35,7 +35,7 @@
 public class AnalyzeBetaIT {
 
   private static final String PROJECT_ID = System.getenv("GOOGLE_CLOUD_PROJECT");
-  private static final String BUCKET = PROJECT_ID;
+  private static final String BUCKET = "cloud-samples-data";
```
Comment (averikitsch) [inline language/analysis/src/test/java/com/google/cloud/language/samples/AnalyzeBetaIT.java]: Do you need this if you are hard-coding the link to the bucket?

Comment (nnegrey) [reply language/analysis/src/test/java/com/google/cloud/language/samples/AnalyzeBetaIT.java]: Nope

**Comment by averikitsch** [comment]
Thanks for the update! @lesv can you approve this PR?

**Pull request merged**